### DISCUSSION TITLE
Timeline of Celebrities who Supported this movement

### DISCUSSION THREAD

**Discussion by ziXet** [WeAreMahsaAmini/Core#201] [Ideas]
Hi,

I believe it is important to remember who joined this movement at which time/stage.

Just created this example fiddle to express my idea visually.
https://jsfiddle.net/hczpv6Ld/1/

We can add the important event in there too. for example celebrity X joined the movement after event Y.

This can be published in the "wearemahsaamini.com".

Let me know what do you think.

**Comment by AnAnonymousOne** [comment]
This is a very helpful information for us to remember but I don’t think this would help more celebrities to come around the idea of supporting people as doing this would suggest that we will hold that against them. I think it may cause them not to respond at all because they would think that it wouldn’t matter anymore. 
I think one of the main reasons that we are doing this, is to encourage more celebrities to support the people.

**Reply by AnAnonymousOne** [reply]
I’m of course talking about Iranian celebrities. I don’t know if that was what you mean or not.

**Reply by ziXet** [reply]
Yeah. I meant Iranian celebrities. thanks for the feedback. lets see what other people say about the idea.
